support-frontend/assets/components/paypalExpressButton/PayPalExpressButton.jsx

```diff
@@ -24,61 +26,48 @@ type PropTypes = {|
   amount: number,
   billingPeriod: BillingPeriod,
   setupRecurringPayPalPayment: SetupPayPalRequestType,
+  updatePayPalButtonReady: (boolean) => Action,
 |};
 
+const tokenToAuthorisation = (token: string): PayPalAuthorisation => ({
+  paymentMethod: PayPal,
+  token,
+});
 
-// Q. Why is this a class rather than a function?
```

Comment:
For the record - we decided this was an unnecessary optimisation given the complexity involved in holding onto the same iframe (which is now more fragile with preact 10)